client/components/whats-new/index.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Card } from '@automattic/components';
+/**
+ * Internal Dependencies
+ */
+import QueryWhatsNewList from 'calypso/components/data/query-whats-new';
+import { getWhatsNewList } from 'calypso/state/whats-new/selectors';
+function WhatsNew( { list, isListReady } ) {
+	return (
+		<Card>
+			<div>
+				{ ! isListReady && <QueryWhatsNewList /> }
```

Comment:
You can always render `<QueryWhatsNewList />` here without any conditional. The component itself will make sure the list is fetched only on mount.